Jenkinsfile

```diff
@@ -286,6 +288,18 @@ pipeline {
         tag pattern: 'v\\d+\\.\\d+\\.\\d+', comparator: 'REGEXP'
       }
       stages {
+        stage('Docker build') {
```

Comment:
If the docker registry instance is only for internal usage/access, what do you think to change this stage with 'Generate staging docker image' or something similar?

Reply:
The goal that I believe @eyalkoren is hoping for is to have a Docker image that can be consumed by the public and is not just only for internal usage.

Reply:
Yes, it needs to be public. We are going to publish a blog post next week telling users how it can be used as an init container in Kubernetes.

Reply:
got it, what about 'Docker image' or `Docker push` as it does consist of two steps, `docker build` and `docker push`, and the aim of this stage is to publish the Docker image,?

Reply:
I changed it to `Docker push` per your latest suggestion. Thanks!